source/dpp/runtime/app.d

```diff
@@ -11,22 +11,23 @@ import dpp.from;
 void run(in from!"dpp.runtime.options".Options options) @safe {
     import std.stdio: File;
     import std.exception: enforce;
-    import std.process: execute;
+    import std.process: spawnProcess, wait;
     import std.array: join;
     import std.file: remove;
+    import std.conv : text;
 
     foreach(dppFileName; options.dppFileNames)
         preprocess!File(options, dppFileName, options.toDFileName(dppFileName));
 
     if(options.preprocessOnly) return;
 
     const args = options.dlangCompiler ~ options.dlangCompilerArgs;
-    const res = execute(args);
-    enforce(res.status == 0, "Could not execute `" ~ args.join(" ") ~ "`:\n" ~ res.output);
```

Comment:
no need to have this, if the target compiler can handle what's been given then we just pass it through.

Reply:
It's needed or else the compiler failing would cause d++ to return 0.

Reply:
sorry, I commented in the wrong place. This line is still there, just moved down a bit.